doc/api/repl.markdown

```diff
@@ -237,3 +237,27 @@ The following key combinations in the REPL have these special effects:
   - `<ctrl>D` - Similar to the `.exit` keyword.
   - `<tab>` - Show both global and local(scope) variables
 
+
+### Customizing Object displays in the REPL
+
+The REPL module internally uses
+[util.inspect()][], when printing values. But, `util.inspect` delegates the call
```

Comment:
I'd recommend using However instead of But here. It's slightly more formal english.